develop/dev-guide-bookshop-schema-design.md

```diff
@@ -114,6 +115,14 @@ On the database details page of TiDB Cloud, click the **Import** button to enter
     arn:aws:iam::494090988690:role/s3-tidb-cloud-developer-access
     ```
 
+    ![The fill from step of import data](/media/develop/import-data-fill-from.png)
```

Comment:
The UI is supposed to be changed today for a feature, which makes the screenshot outdated. Could we not provide the screenshot?